Procfile

```diff
@@ -1,2 +1,2 @@
-web: ./modules/service/target/universal/stage/bin/lucuma-odb-service -Dfile.encoding=UTF-8 -Dcats.effect.stackTracingMode=DISABLED -Dcats.effect.tracing.mode=none serve
-calibration: ./modules/calibrations/target/universal/stage/bin/calibrations-service -Dfile.encoding=UTF-8 -Dcats.effect.stackTracingMode=DISABLED -Dcats.effect.tracing.mode=none
+web: ./modules/service/target/universal/stage/bin/lucuma-odb-service -Dfile.encoding=UTF-8 -J-javaagent:./modules/service/target/universal/stage/lib/com.newrelic.agent.java.newrelic-agent-8.19.0.jar -J-Dnewrelic.config.file=newrelic.yml serve
```

Comment:
I'm unsure if this actually works. Regardless, the hardcoded version is less than ideal